src/main/java/com/configcat/SegmentCondition.java

```diff
@@ -0,0 +1,30 @@
+package com.configcat;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Describes a condition that is based on a segment.
+ */
+public class SegmentCondition {
+    /**
+     * The index of the segment that the condition is based on.
```

Comment:
I'd rather move these docs comments from private fields to the public accessors.